docs/api-markdown-documenter/index.js

```diff
@@ -4,16 +4,30 @@
  */
 
 const chalk = require("chalk");
-
+const versions = require("../data/versions.json");
 const { renderApiDocumentation } = require("./render-api-documentation");
 
-renderApiDocumentation().then(
-	() => {
-		console.log(chalk.green("API docs written!"));
-		process.exit(0);
-	},
-	(error) => {
-		console.error("API docs could not be written due to an error:", error);
-		process.exit(1);
-	},
-);
+const renderMultiVersion = process.argv[2];
+
+// Turn on 1.0 only for now
+docVersions = renderMultiVersion
+	? versions.params.currentVersion.concat(versions.params.previousVersions)
+	: versions.params.previousVersions;
+
+const apiDocRenders = [];
+
+docVersions.forEach((version) => {
+	apiDocRenders.push(
+		renderApiDocumentation(version).then(
+			() => {
+				console.log(chalk.green(`API ${version} docs written!`));
+				process.exit(0);
+			},
+			(error) => {
+				console.error(`API ${version} docs could not be written due to an error:`, error);
+				process.exit(1);
+			},
+		),
+	);
+});
+Promise.all(apiDocRenders);
```

Comment:
This needs to be awaited.
```suggestion
await Promise.all(apiDocRenders);
```